extract-metadata.py

```diff
@@ -0,0 +1,71 @@
+from dandi.dandiapi import DandiAPIClient
+from dandi.dandiapi import RemoteDandiset
+import numpy as np
```

Comment:
`numpy` seems to never be used in this script -- probably can remove the import

Reply:
Good catch, removed.